source/fundamentals/connection/connect.txt

```diff
@@ -95,6 +95,14 @@ verify that the connection is successful:
 .. literalinclude:: /code-snippets/connection/stable-api-connect.js
    :language: javascript
 
+.. note::
+
+   As of v4.7, the {+driver-short+} will automatically call the ``MongoClient.connect()``
+   method when using the client to interact with your MongoDB instance, **except** when
```

Comment:
S: the sentence is a little long, and the "except" clause seems pretty important. better as its own sentence?